contracts/mixnet/src/storage.rs

```diff
@@ -48,6 +51,46 @@ pub fn config_read(storage: &dyn Storage) -> ReadonlySingleton<State> {
     singleton_read(storage, CONFIG_KEY)
 }
 
+fn reward_pool(storage: &dyn Storage) -> ReadonlySingleton<Uint128> {
+    singleton_read(storage, REWARD_POOL_PREFIX)
+}
+
+pub fn mut_reward_pool(storage: &mut dyn Storage) -> Singleton<Uint128> {
+    singleton(storage, REWARD_POOL_PREFIX)
+}
+
+pub fn reward_pool_value(storage: &dyn Storage) -> Uint128 {
+    match reward_pool(storage).load() {
+        Ok(value) => value,
+        Err(_e) => Uint128(INITIAL_REWARD_POOL),
+    }
+}
+
+#[allow(dead_code)]
```

Comment:
Why is it a `dead_code`. is it not the case we should be incrementing reward pool at some points?